src/services/auth.ts

```
@@ -52,18 +44,26 @@
         throw new UnauthorizedError(KnownErrorCode.BANNED_USER);
       }
 
+      const required_roles = options?.roles ?? [];
+      const user_roles = await this.getUserRoles(user.id);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)